system_test/aest_sync_SUITE.erl

```diff
@@ -444,9 +454,14 @@ sync_speed(Cfg) ->
 
 node_name(I) -> list_to_atom("node" ++ integer_to_list(I)).
 
-get_block(NodeName, Height, Cfg) ->
-    Query = #{height => Height},
-    case http_get(NodeName, ext_http, [v2, 'block-by-height'], Query, Cfg) of
+get_block(NodeName, Height, _Cfg) ->
+    case request(NodeName, 'GetBlockByHeight', #{height => Height}) of
         {ok, 200, Block} -> Block;
         {ok, 404, _} -> undefined
     end.
+
+request(Node, Id, Params) ->
```

Comment:
This is a new definition of request, but where was the old one? I expected it to be removed?

Reply:
Ah, it is `imported`, that import should be removed...

Reply:
Thanks missed the import.